sdk/appconfiguration/app-configuration/src/appConfigCredential.ts

```diff
@@ -10,6 +10,12 @@ export class AppConfigCredential implements ServiceClientCredentials {
     this.secret = secret;
   }
 
+  /**
+   * Signs a request with the values provided in the credential and secret parameter.
+   *
+   * @param {WebResource} webResource The WebResource to be signed.
+   * @returns {Promise<WebResource>} The signed request object.
```

Comment:
I think we might want to mark this method with the `@internal` and `@ignore` tags. It probably has to be a public method since it's used by core-http at some point, but we wouldn't expect customers to call it. Those flags will prevent the method from showing up in our documentation.

Reply:
As discussed, actually the whole class should be marked internal since it isn't exported from the package, only from this file for internal use.

Reply:
Done.